configs/katello/3.10.yaml

```diff
@@ -0,0 +1,83 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.10.1:
+    :redmine_version_id: 919
+:releases:
+  :3.10.0:
+    :redmine_version_id: 937
+:code_name:
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.10
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-installer:
+    :branch: KATELLO-3.10
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/1.20
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 0.14.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.3.5
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :bastion:
+    :branch: 6.1.12
+    :repo: https://github.com/Katello/bastion.git
+  :puppet-pulp:
```

Comment:
Could you sort these? makes it easier to verify.

I also miss puppet-katello. The procedure omitted it as well.